### PR TITLE
Add in support for the Bill and related models

### PR DESCRIPTION
**Pull request by nickhammond**

### CODE REVIEW

File: lib/quickeebooks/common/service_crud.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  def list(filters = [], page = 1, per_page = 20, sort = nil, options = {})
+    fetch_collection(model, filters, page, per_page, sort, options)
+  end
+
+  def included_model
+    self.class.to_s.split('::').last
+  end 
+
+  def model
```
Comment (ruckus) [inline lib/quickeebooks/common/service_crud.rb:48]: There will be issues here if this module gets mixed into any `Windows` classes, right? Maybe we can break this one method out into another mixin which does the distinct work of referencing `Online` or `Windows`

**Comment by ruckus** [comment]
Wow, this is awesome. Thank you! I just created a note in the `ServiceCRUD` mixin.

File: lib/quickeebooks/common/service_crud.rb (hunk 1 above)
Comment (nickhammond) [inline lib/quickeebooks/common/service_crud.rb:48]: Good catch, yah that'll should probably be extracted out.

Comment (minimul) [inline lib/quickeebooks/common/service_crud.rb:48]: Actually, self.class returns all the namespacing from where the module is mixed in so I approached it from that angle. Updated the pull.

**Comment by minimul** [comment]
Note: this pull is passing on Travis now (REE, 1.9.2, 1.9.3). https://travis-ci.org/nickhammond/quickeebooks/builds

**Comment by ruckus** [comment]
Awesome. Thank you!

**Pull request merged**